javalib/src/main/scala/java/util/Optional.scala

```diff
@@ -0,0 +1,56 @@
+package java.util
+
+class Optional[T] private (value: T) {
+
+  def get(): T = {
+    if (!isPresent())
+      throw new NoSuchElementException()
+    else value
```

Comment:
Both branches an `if` should agree on whether they're on new line or not. So this should be:
```scala
if (!isPresent())
  throw new NoSuchElementException()
else
  value
```
Alternatively, it could be:
```scala
if (!isPresent())
  throw new NoSuchElementException()
value
```
(without `else`)